rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -31,6 +31,7 @@ PConjunction. Proc14 ::= Proc14 "/\\" Proc15 ;
 PDisjunction. Proc13 ::= Proc13 "\\/" Proc14 ;
 PEval.        Proc12 ::= "*" Name ;
 PMethod.      Proc11 ::= Proc11 "." Var "(" [Proc] ")" ;
+PExprs.       Proc11  ::= "(" Proc4 ")" ;
```

Comment:
I think that this makes the grammar ambiguous. It would be better to solve this some other way.

Reply:
Why ? `PExprs` is of higher precedence than other *parentheses using categories* and the grammar has no way of being ambiguous ATM.

Reply:
And current change is as least invasive as it can be. There's no need for `(1,)` , tuples are created in the same way as before `(1,2,3,4)` and it allows for using multiple nested expressions using parentheses `(1  + (2 * ( 4 / 2)))` . Unless you don't have any examples when it breaks I think this is a good solution.

Reply:
@guardbotmk3 does your intuition tell you this won't work or you have an example that breaks the grammar?

Reply:
@guardbotmk3 This one is still on you. Can you explain or provide a counter example to why it won't work?